py/bitbox02/README.md

```diff
@@ -15,3 +15,16 @@ bitbox02.
 In this folder is the Python API for communicating with the BitBox02 device.
 
 The folder `generated` contains files generated by `python-protobuf`. Regenerate with `make`.
+
+
+### Development environment
+
+It is highly recommended that you use the dockerized setup while developing, guide for setting it up
+can be found in [BUILD.md](../../BUILD.md).
+
+If you intended to develop instead on your local environment be vary that some development procedures
+such as regenerating files have their libraries NOT in requirements.txt as they are not always necessary to run python
+scripts.
```

Comment:
suggestionn:

not always necessary to run the python scripts => not necessary for the bitbox02 Python package.

They are never necessary for the bitbox02 package, as these are dev-dependencies not relevant for the package users.